source/ways-of-working/path-to-live/application-approval-production.html.md.erb

```diff
@@ -0,0 +1,41 @@
+---
+title: Application approval into Production
+last_reviewed_on: 2021-06-21
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+This page explains the typical processes an application team must ensure they/their app has been through before getting the green light to deploy to production
+
+1. OAT - Operational acceptance testing covers multiple topics to try and ensure that your application is ready to go into production, it evidences topics such as best practices, avoiding anti-patterns, scalability, disaster recovery etc. 
```

Comment:
not really sure if best practices and avoiding anti-patterns should be included in here too vague, is that actually included as well?

```suggestion
1. OAT - Operational acceptance testing covers multiple topics to ensure that your application is ready to go into production, it include topics such as scalability, disaster recovery, logging, best practices, avoiding anti-patterns. 
```

Reply:
Is it worth including the OAT confluence link here?